lib/core/lib/generators/ros/model/model_generator.rb

```diff
@@ -10,7 +10,7 @@ def create_files
       create_file "app/models/#{name}.rb", <<~FILE
```

Comment:
Also this. Why are we generating models when rails already does that?

Reply:
I didn't use the default `invoke(:model)` as even after creating the files, we still have to inject new code into them using the `insert_into_file` thor method. I thought it would be cleaner to just create new files from scratch.

I explained my thoughts in the description of [https://github.com/rails-on-services/ros/pull/67](This PR) actually.